tsconfig.json

```diff
@@ -1,29 +1,26 @@
 {
+    "files": [
+        "index.ts"
+    ],
     "compilerOptions": {
         "target": "es5",
-        "module": "umd",
+        "module": "es2015",
+        "lib": ["es6", "es2015", "dom"],
         "moduleResolution": "node",
-        "isolatedModules": false,
-        "jsx": "react",
         "experimentalDecorators": true,
         "emitDecoratorMetadata": true,
-        "declaration": false,
+        "declaration": true,
         "noImplicitAny": true,
-        "noImplicitUseStrict": false,
         "removeComments": true,
-        "noLib": false,
-        "preserveConstEnums": true,
         "suppressImplicitAnyIndexErrors": true,
-        "sourceMap": true,
+        "inlineSourceMap": true,
```

Comment:
Yes please. Having inline source maps bloats up the file in the situation when the user isn't using source maps. And some tools might not work well with inline source maps.